pkg/dispers/conceptindexor.go

```diff
@@ -107,16 +102,14 @@ func getSalt(concept string) (string, error) {
 
 func hash(str string, salt string) (string, error) {
 
-	// scrypt the cleartext string with the same parameters
-	other, err := scrypt.Key([]byte(str), []byte(salt), defaultN, defaultR, defaultP, defaultDkLen)
-	if err != nil {
-		return "", err
-	}
-	return string(other), err
+	h := sha256.New()
+	h.Write([]byte(salt))
+	hash := h.Sum([]byte(str))
+
+	return string(hash), nil
 }
 
-// isConceptExisting is not useful to create a concept
-// isConceptExisting is useful to test CreateConcept or DeleteConcept
+// isSaltExisting is useful to test CreateConcept or DeleteConcept
 func isConceptExisting(concept string) (bool, error) {
```

Comment:
The method name and the comment do not match. You IDE should warn you about errors like this, what do you currently use?

Reply:
That's weird. I have no comment at all for this method in my local repository. And it's up to date with the remote repo.

I'm using Atom with some add-ons:
- todo
- go-signature-statusbar
- go-plus
- go-debug

I have got a lint to that warn me for this kind of mistake.

Reply:
Ok I think I reviewed on a previous commit and this line has been since deleted, my bad